tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -938,6 +938,35 @@ Tensor normalize_hw(const Tensor& y,const MemoryConfig& output_mem_config) {
    return operation::decorate_as_composite(__func__, _normalize)(y,output_mem_config);
 }
 
+
+using HWFunctionT = std::function<Tensor (const Tensor& y, const MemoryConfig&)>;
+Tensor _make_global_from_hw_impl(HWFunctionT fn, const Tensor& y, const MemoryConfig& output_mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG) {
+    const Shape s_orig = y.shape();
+    TT_FATAL(s_orig.rank() == 4, "Cannot support non-rank 4 Tensor");
+
+    //format to HW
+    Tensor y_chw = reshape(y,1,s_orig[1],s_orig[2],-1);
+    Tensor y_hw = reshape(y_chw,1,1,s_orig[2],-1);
```

Comment:
Could you explain why two reshapes are needed here, and similarly for reformatting the output?

Reply:
What are the limitations/issues with running reshape only once instead of twice?